ci/cscs-ci.yml

```
@@ -127,7 +127,8 @@ build_cscs_amd_rocm:
     - export NOX_SESSION_ARGS="${VARIANT:+($VARIANT}${SUBVARIANT:+, $SUBVARIANT}${DETAIL:+, $DETAIL}${VARIANT:+)}"
     - cd "${WORKDIR}/gt4py" && uv run --script noxfile.py -s "test_${SUBPACKAGE}-${PY_VERSION}${NOX_SESSION_ARGS}"
 
-test_cscs_gh200:
+# TODO(edopao): CI on santis was temporarily disabled because of vCluster issue
+.test_cscs_gh200:
```

```suggestion
test_cscs_gh200:
```
I discussed this with Enrique last time the system was offline and the conclusion was, we don't temporarily switch of unavailable systems.